src/v/rpc/demo/client.cc

```diff
@@ -52,8 +52,6 @@ void cli_opts(boost::program_options::options_description_easy_init o) {
 }
 
 struct load_gen_cfg {
-    load_gen_cfg() = default;
-    load_gen_cfg(const load_gen_cfg&) = default;
```

Comment:
This has now gained move, which is "incorrect" due to the pointer, but I guess lifetime is guaranteed anyway.

Reply:
seems to have the same semantics as before?

Reply:
The copy constructor would have disabled move. But yeah, copying was no safer.

Reply:
I'm wrong.